frontend/src/lib/components/ChartFilter/ChartFilter.tsx

```diff
@@ -10,17 +10,19 @@ import {
     PieChartOutlined,
     TableOutlined,
 } from '@ant-design/icons'
-import { ChartDisplayType, FilterType, ViewType } from '~/types'
+import { ChartDisplayType, FilterType, FunnelVizType, ViewType } from '~/types'
+import { preflightLogic } from 'scenes/PreflightCheck/logic'
 
 interface ChartFilterProps {
     filters: FilterType
-    onChange: (chartFilter: ChartDisplayType) => void
+    onChange: (chartFilter: ChartDisplayType | FunnelVizType) => void
```

Comment:
I think the addition of this type just confuses things, especially if they're going to be used interchangeably. My preferred approach would be to put everything in `ChartDisplayType` and do away with `FunnelVizType`, OR have e.g. `ChartDisplayType.FUNNELS` cover everything and have `FunnelVizType` make it more specific. 

What do you think @mariusandra @alexkim205 ? I don't have a strong opinion here but feel like we should agree on a convention

Reply:
Scratch that, I really think `FunnelVizType` should be removed. We already have `ViewType.FUNNELS` to describe funnels overall.

Reply:
that's where the typing will come from for `funnel_viz_type`, see this comment: https://github.com/PostHog/posthog/pull/5133#discussion_r670820614

Reply:
To clarify, `FunnelVizType` is the type of funnel visualization (since that's separate from overarching visualizations (like Trends or Funnels) )

Although, valid question whether ChartFilterProps, the top level interface should expect top level visualisation type or specific-to-funnels visualization type as well.

Reply:
Right, @neilkakkar , `ViewType` includes Trends, Stickiness, Funnels, Retention, etc. and corresponds to overall type of query.

Then `ChartDisplayType` describes, for instance, linear line graph, cumulative line graph, bar chart, pie, etc. and seems like the logical place to distinguish between time to convert, steps, and historical trends as those are "types of visualization".

Reply:
I think it's also a little confusing to reuse `ChartDisplayType` for something like time conversion because that technically uses the `ActionsLineGraphLinear` display type and would also probably cause confusion on the frontend if we checked for `insights === FUNNELS && display === ChartDisplayType.ActionsLineGraphLinear` vs just `funnel_viz_type === FunnelVizType.Trends/TimeConversion/whatever`

Reply:
but also agree we should make our chart/display type system easier to navigate in general :o